### DISCUSSION TITLE
ISSUE with setting up foundry

### DISCUSSION THREAD

**Discussion by olahamid** [Cyfrin/foundry-full-course-cu#517] [Q&A] [answered]
when i hit the "forge build"command i got an error and it won't compile.
BASH ERROR CODE
```gitpod /workspace/new_folder2/foundry-f233/foundry-simple-storage-f233 (master) $ forge build
[⠊] Compiling...
[⠒] Unable to resolve imports:
      "../src/Counter.sol" in "/workspace/new_folder2/foundry-f233/foundry-simple-storage-f233/test/Counter.t.sol"
with remappings:
      ds-test/=/workspace/new_folder2/foundry-f233/foundry-simple-storage-f233/lib/forge-std/lib/ds-test/src/
      forge-std/=/workspace/new_folder2/foundry-f233/foundry-simple-storage-f233/lib/forge-std/src/
[⠘] Installing solc version 0.8.19
[⠊] Successfully installed solc 0.8.19
[⠢] Compiling 22 files with 0.8.19
[⠆] Solc 0.8.19 finished in 105.04ms
Error: 
Compiler run failed:
Error (6275): Source "src/Counter.sol" not found: File not found. Searched the following locations: "/workspace/new_folder2/foundry-f233/foundry-simple-storage-f233".
 --> test/Counter.t.sol:5:1:
  |```

**Comment by 0xusmanf** [comment]
Make sure you are writing the correct file name in your import path in `Counter.t.sol` file at line `5`. Check for spelling mistakes. Your file in `src` folder should be named `Counter.sol`, if your import path is `"../src/Counter.sol"`.

**Reply by olahamid** [reply, marked as answer]
Thanks alot usman, I just deleted the test file and it worked.

**Reply by 0xusmanf** [reply]
You are welcome.

**Comment by TSHEPO-CLOUD** [comment]
Good day @olahamid, am not sure which lesson you on but i would suggest you go through the first 20 minutes of lesson 7 and see if it might help.

**Reply by olahamid** [reply]
I'm on lesson 6(compiling in foundry), I will check out first 20 minute of lesson 7 hopefully it works. Thanks

**Reply by 0xusmanf** [reply]
It is not an issue with Foundry, it's an import issue. Check the comment above.

**Reply by Joel-eth** [reply]
sorry please but how do I delete the test file?

**Reply by 0xusmanf** [reply]
Just right-click on it and hit delete.
![2024-04-19](https://github.com/Cyfrin/foundry-full-course-f23/assets/110072406/8bfee5d9-93fc-4a6c-9268-11edcf54dc15)
